### PR TITLE
charset bug fixed

### PR DESCRIPTION
**Pull request by muhammetus**

### CODE REVIEW

File: src/VCard.php [hunk 1]
```diff
@@ -330,9 +336,9 @@ public function buildVCalendar()
      * @return string decoded
      * @param  string $value The value to decode
      */
-    private function decode($value)
+    private function decode($value) 
     {
-        return htmlspecialchars_decode((string) $value, ENT_QUOTES);
+        return htmlspecialchars_decode((string) iconv($this->charset,"ISO 8859-9",$value), ENT_QUOTES);
```
Comment (jeroendesloovere) [inline src/VCard.php]: Is it possible to add a space after every "," in the iconv function? Thanks.

File: src/VCard.php [hunk 2]
```diff
@@ -330,9 +336,9 @@ public function buildVCalendar()
      * @return string decoded
      * @param  string $value The value to decode
      */
-    private function decode($value)
+    private function decode($value)
```
Comment (jeroendesloovere) [inline src/VCard.php]: Please remove redundant space.

File: src/VCard.php [hunk 3]
```diff
@@ -17,7 +17,7 @@
  * @author Jeroen Desloovere <info@jeroendesloovere.be>
  */
 class VCard
-{
+{
```
Comment (jeroendesloovere) [inline src/VCard.php]: Can you remove this space?

File: src/VCard.php (hunk 1 above)
Comment (yuks) [inline src/VCard.php]: I think it's not necessary to create a new method for this issue. Above solution is only for turkish letters. I had the same charset bug for german letters and fixed only by adding utf8_decode() function like this:

return htmlspecialchars_decode((string) utf8_decode($value), ENT_QUOTES);

**Pull request merged**